api/models/eplant2.py

```diff
@@ -49,3 +49,29 @@ class Publications(db.Model):
     journal: db.Mapped[str] = db.mapped_column(db.String(64), nullable=False, primary_key=True)
     title: db.Mapped[str] = db.mapped_column(TEXT(), nullable=False, primary_key=True)
     pubmed: db.Mapped[str] = db.mapped_column(db.String(16), nullable=False, primary_key=True)
+
+
+class TAIR10_GFF3(db.Model):
```

Comment:
Rename to TAIR10GFF3

Reply:
@TeresasaZ Follow Asher's request about DB names, following the name conventions you see in BAR_API